api/audit/migrations/0007_auditlog_is_system_event.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.16 on 2022-11-08 12:04
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('audit', '0006_auditlog_master_api_key'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='auditlog',
+            name='is_system_event',
```

Comment:
We should discuss this with @kyle-ssg. We've added this field to the AuditLog so that we can track events that happen which are not performed by a specific user. @gagantrivedi we should also add a query parameter to the AuditLogViewSet so that they can be included / excluded from the list?

Reply:
Yep, done